packages/opentelemetry-exporter-prometheus/src/prometheus.ts

```diff
@@ -69,10 +68,10 @@ export class PrometheusExporter implements MetricExporter {
       config.endpoint || PrometheusExporter.DEFAULT_OPTIONS.endpoint
     ).replace(/^([^/])/, '/$1');
 
-    if (config.startServer || PrometheusExporter.DEFAULT_OPTIONS.startServer) {
+    if (config.startServer) {
       this.startServer(callback);
-    } else if (callback) {
-      callback();
+    } else {
+      callback?.();
```

Comment:
Please call this asynchronously or you risk [releasing zalgo](https://blog.izs.me/2013/08/designing-apis-for-asynchrony)

Reply:
👍 will do.